src/test/java/com/thealgorithms/maths/FindMinTest.java

```diff
@@ -12,13 +12,14 @@
 public class FindMinTest {
 
     @ParameterizedTest
-    @MethodSource("provideStringsForIsBlank")
+    @MethodSource("inputStream")
     void numberTests(int expected, int[] input) {
         Assertions.assertEquals(expected, FindMin.findMin(input));
     }
 
-    private static Stream<Arguments> provideStringsForIsBlank() {
-        return Stream.of(Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}), Arguments.of(5, new int[] {5, 5, 5, 5, 5}), Arguments.of(0, new int[] {0, 192, 384, 576}), Arguments.of(-1, new int[] {-1, 2, 5, 10}), Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}));
+    private static Stream<Arguments> inputStream() {
+        return Stream.of(Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}), Arguments.of(5, new int[] {5, 5, 5, 5, 5}), Arguments.of(0, new int[] {0, 192, 384, 576}), Arguments.of(-1, new int[] {-1, 2, 5, 10}), Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}),
```

Comment:
```suggestion
        return Stream.of(
            Arguments.of(1, new int[] {1, 2, 3, 4, 5, 6, 7, 8, 9, 10}),
            Arguments.of(5, new int[] {5, 5, 5, 5, 5}),
            Arguments.of(0, new int[] {0, 192, 384, 576}),
            Arguments.of(-1, new int[] {-1, 2, 5, 10}),
            Arguments.of(-10, new int[] {-10, -9, -8, -7, -6, -5, -4, -3, -2, -1}),
```
Can we do something like that?

Reply:
No, `clang-format` will complain. The reason is this:
https://github.com/TheAlgorithms/Java/blob/cada67b1eb58558b08049b84ee93a15de7867e4b/.clang-format#L52

Should I make another PR and set it to something like `100` or `80` and reformat all of the files?

Reply:
Hmm, I think no. There should be a better way than using ColumnLimit